pkg/internal/checkup/vmi.go

```diff
@@ -156,3 +164,12 @@ func trafficGenBootCommands(configDiskSerial string) []string {
 		fmt.Sprintf("cp %s/*.py %s", configMountDirectory, trex.StreamsPyPath),
 	}
 }
+
+func vmiUnderTestBootCommands(configDiskSerial string) []string {
+	const configMountDirectory = "/mnt/app-config"
```

Comment:
Please consider increasing the scope of the existing `configMountDirectory` const, and share it between the functions.

Reply:
we don't need it anywhere else. Everything is copied from this dir to where it will actually be executed.